var/spack/repos/builtin/packages/bamtools/package.py

```diff
@@ -37,10 +38,15 @@ class Bamtools(Package):
     version('2.2.3', '6eccd3e45e4ba12a68daa3298998e76d')
 
     depends_on('cmake', type='build')
```

Comment:
`CMakePackage` comes with this dependency automatically, so you can remove this line now.